### ISSUE TITLE
When Refreshing endpoints should also consider removing of the nodes that are removed from SC

### ISSUE THREAD

**Issue by HEskandari** [Particular/ServiceInsight#168]
This can be tested once [deletion / purging of the messages][1] is done in SC but basically when you refresh the Endpoint explorer list, we need to also remove the endpoints that are no longer relevant (probably because messages in those Endpoints are purged).

[1]:https://github.com/Particular/ServiceControl/issues/114
